frontend/packages/console-app/src/components/terminal/__tests__/MinimizeRestoreButton.spec.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Button } from '@patternfly/react-core';
+import { OutlinedWindowMinimizeIcon, OutlinedWindowRestoreIcon } from '@patternfly/react-icons';
+import MinimizeRestoreButton from '../MinimizeRestoreButton';
+
+describe('MinimizeRestoreButton', () => {
+  it('should render button', () => {
+    const wrapper = shallow(
+      <MinimizeRestoreButton minimizedText="Minimize" restoreText="Restore" onClick={() => null} />,
+    );
+    expect(wrapper.find(Button).exists()).toEqual(true);
+  });
+
+  it('should render minimize button when minimized is true', () => {
+    const wrapper = shallow(
+      <MinimizeRestoreButton
+        minimizedText="Minimize"
+        restoreText="Restore"
+        minimized
+        onClick={() => null}
+      />,
+    );
+    expect(wrapper.find(OutlinedWindowMinimizeIcon).exists()).toEqual(true);
+  });
+
+  it('should render restore button when minimized is false', () => {
+    const wrapper = shallow(
+      <MinimizeRestoreButton
+        minimizedText="Minimize"
+        restoreText="Restore"
+        minimized={false}
+        onClick={() => null}
+      />,
+    );
+    expect(wrapper.find(OutlinedWindowRestoreIcon).exists()).toEqual(true);
+  });
+
+  it('should invoke onclose callback with argument true when minimized button clicked and false when restore button clicked', () => {
+    let called: boolean;
+    const onClose = (state: boolean) => {
+      called = state;
+    };
```

Comment:
I think this is already a jest functionality: https://jestjs.io/docs/en/expect#tohavebeencalledwitharg1-arg2-

You're calling it twice, so maybe you'll want this: https://jestjs.io/docs/en/22.x/expect#tohavebeenlastcalledwitharg1-arg2-

Arguably this is two tests though.